ui/src/app/pages/profile/profile-access-modules.tsx

```diff
@@ -0,0 +1,201 @@
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+import {ControlledTierBadge} from 'app/components/icons';
+import {AoU} from 'app/components/text-wrappers';
+import {displayDateWithoutHours, lensOnProps} from 'app/utils';
+import {getRegistrationTasksMap} from 'app/utils/access-utils';
+import {useNavigation} from 'app/utils/navigation';
+import {serverConfigStore, useStore} from 'app/utils/stores';
+import {Profile} from 'generated/fetch';
+import {DataAccessPanel} from './data-access-panel';
+import {ProfileRegistrationStepStatus} from './profile-registration-step-status';
+import {styles} from './profile-styles';
+
+enum RegistrationStepStatus {
```

Comment:
all of this code was migrated from profile-component and then converted to functional style.  Fortunately, this really only involved removing `this.` because it was already stateless.